lib/components/QuizButton.dart

```diff
@@ -1,35 +1,30 @@
 import 'package:flutter/material.dart';
 
-class MyButton extends StatefulWidget {
+class QuizButton extends StatefulWidget {
   final String text;
   final VoidCallback onPress;
-  int keys;
 
-  MyButton(
+  QuizButton(
       {Key key,
       this.text,
-      this.keys,
       this.onPress})
       : super(key: key);
   
   @override
-  _MyButtonState createState() => new _MyButtonState();
+  QuizButtonState createState() => new QuizButtonState();
 }
 
-class _MyButtonState extends State<MyButton>{
-  String _displayText;
+class QuizButtonState extends State<QuizButton>{
+  
 
   initState() {
```

Comment:
Do we still need initState?

Reply:
In case we decide to add something(like animations, etc.) later then initState will come useful.